README.md

```diff
@@ -135,6 +135,7 @@ Before continuing with the installation steps in this README, users of Windows S
   * `sudo apt install libyaml-dev make pkg-config sqlite3 zlib1g-dev libgmp-dev`
   * `sudo apt install libreadline-dev libssl-dev`
 2. Add GPG Key & Install RVM:
+  * `sudo apt install gnupg2`
```

Comment:
Thanks for opening this @RealRichi3, its looking good. Maybe we should add some text before the command smth like "Install gnupg2 if you haven't already: "  What do you think?

Reply:
True, I'll add that

Reply:
Or better still, since I've already contributed, let me make it a `first-time-issue` so others can fix it

Reply:
Sounds great  :+1: . Thanks